modin/pandas/utils.py

```diff
@@ -238,5 +252,49 @@ def check_both_not_none(option1, option2):
     return not (option1 is None or option2 is None)
 
 
+def _doc_binary_op(
+    operation, bin_op, other="right", returns="Series", primary_object="Series"
+):
+    """
+    Return callable documenting `Series` or `DataFrame` binary operator.
+
+    Parameters
+    ----------
+    operation : str
+        Operation name.
+    bin_op : str
+        Binary operation name.
+    other : str, default: 'right'
+        The second operand name.
+    returns : str, default: 'Series'
+        Type of returns.
+    primary_object : str, default: 'Series'
+        The primary object to document.
+
+    Returns
+    -------
+    callable
+    """
+    if primary_object == "Series":
+        other_type = "Series or scalar value"
+    elif primary_object == "BasePandasDataset":
+        other_type = "BasePandasDataset or scalar value"
+    else:
+        raise NotImplementedError(
```

Comment:
What about DataFrame?

Reply:
`_doc_binary_op` is not used for `DataFrame`, thus since we don't use it, i think it's not reasonable to support unused code.

Reply:
Why do we not use it for DataFrame? I wonder if somebody added it to a DataFrame method, he would get the error and be confused. Maybe we should support this now?

Reply:
`_doc_binary_op` is used to document dunder binary methods which are completely inherited from the `BasePandasDataset` base class and doesn't overridden in the `DataFrame`.
Ok, added support for `DataFrame` `left`.